network/server.go

```diff
@@ -758,26 +754,22 @@ func (s *Server) SubscribeCh() (<-chan *peerEvent.PeerEvent, error) {
 func (s *Server) updateConnCountMetrics(direction network.Direction) {
 	switch direction {
 	case network.DirInbound:
-		s.metrics.InboundConnectionsCount.Set(
-			float64(s.connectionCounts.GetInboundConnCount()),
-		)
+		metrics.SetGauge([]string{"inbound_connections_count"}, float32(s.connectionCounts.GetInboundConnCount()))
+
 	case network.DirOutbound:
-		s.metrics.OutboundConnectionsCount.Set(
-			float64(s.connectionCounts.GetOutboundConnCount()),
-		)
+		metrics.SetGauge([]string{"outbound_connections_count"}, float32(s.connectionCounts.GetInboundConnCount()))
```

Comment:
```suggestion
		metrics.SetGauge([]string{"outbound_connections_count"}, float32(s.connectionCounts.GetOutboundConnCount()))
```
copypasta error maybe?

Reply:
Indeed. Good catch.